pkgs/applications/version-management/git-and-tools/conform/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "conform";
+  version = "0.1.0-alpha.25";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "siderolabs";
+    repo = "conform";
+    sha256 = "sha256-WgWgigpqPoEBY4tLjbzK02WFwrCWPGQWJ5eakLv5IWw=";
+  };
+
+  vendorSha256 = "sha256-Oigt7tAK4jhBQtfG1wdLHqi11NWu6uJn5fmuqTmR76E=";
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Policy enforcement for your pipelines.";
```

```suggestion
    description = "Policy enforcement for your pipelines";
```

contributing mandates no period at the end of description